.github/ISSUE_TEMPLATE/1-bug-report.md

```
@@ -0,0 +1,38 @@
+  2. Click on '....'
+  3. Scroll down to '....'
+  4. See error
+  
+  **Expected behavior**
+  A clear and concise description of what you expected to happen.
+  
+  **Screenshots**
+  If applicable, add screenshots to help explain your problem.
+  
+  **Desktop (please complete the following information):**
+  - OS: [e.g. iOS]
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)